src/app/content/components/Page/PageContent.tsx

```diff
@@ -21,6 +21,15 @@ export const contentTextStyle = css`
 // Search and key term highlights need to target .math as well,
 // otherwise math elements won't have full height background color
 const SELF_AND_CHILD_MATH_SELECTOR = '&, & .math';
+const screenreaderOnly = `
```

Comment:
This might be nice to have in a generic styles or utils file, I can see us reusing it

Reply:
Yeah, I should probably make that an export from some util set.